link-verifier/fileTests/goodFiles/fileWithLowercasemdIntheName.md

```diff
@@ -3,3 +3,9 @@
 # Here's a random link for it to test as well
 [verify-links.py](../../verify-links.py)
 [CI-CD-Github-Actions](https://github.com/FreeRTOS/CI-CD-Github-Actions)
+# Test that it will find this url
+https://www.google.com
+# Test that it will find this url and drop the slash
+https://www.google.com/
+# Test that it will find this url by dropping the coma
+https://www.google.com,
```

Comment:
Test to make sure we don't try and use the trailing coma as part of the URL, currently an issue with the checker.